### PR TITLE
Fix system pod startup and enabled scaled ep tests

### PR DESCRIPTION
**Pull request by phudtran**
This PR does the following:

- Enable Scaled endpoint tests
- Fix an issue where system pods fail to start due to agent RPCs being called before the agent is loaded.

### CODE REVIEW

File: mizar/common/rpc.py [hunk 1]
```diff
@@ -212,6 +212,8 @@ def update_agent_metadata(self, ep):
         cmd = f'''{self.trn_cli_update_agent_metadata} -i \'{itf}\' -j \'{jsonconf}\''''
         logger.info("update_agent_metadata: {}".format(cmd))
         returncode, text = run_cmd(cmd)
+        if "error" in text:
```
Comment (vinaykul) [inline mizar/common/rpc.py]: Is this the fix to catch the race condition where update hits before agent load?

Comment (phudtran) [reply mizar/common/rpc.py]: Yeah, the return code always returns none, but the output text will have "error" when something goes wrong.
I can turn this into a constant.

File: teste2e/common/k8s.py [hunk 2]
```diff
@@ -147,7 +147,7 @@ def pod_exec_stdout(self, name, cmd):
                       stdout=True, tty=False)
         return resp
 
-    def create_service(self, name):
+    def create_service(self, name, vpc="vpc0", subnet="net0"):
```
Comment (vinaykul) [inline teste2e/common/k8s.py]: Constants.py default_ep_vpc / default_ep_subnet ?

File: teste2e/common/k8sservice.py [hunk 3]
```diff
@@ -20,6 +20,7 @@ def eptype(self):
     def backends(self):
         return set(self.endpoints.keys())
 
-    def add_endpoint(self, name):
-        ep = self.api.create_pod(name, scaledep=self.name)
+    def add_endpoint(self, name, vpc_name="vpc0", subnet_name="net0"):
```
Comment (vinaykul) [inline teste2e/common/k8sservice.py]: same here. from constants default_ep_vpc/net?

**Review by vinaykul — APPROVED** [review]
A couple of minor fix suggestions.
LGTM.

File: teste2e/common/k8sservice.py (hunk 3 above)
Comment (phudtran) [reply teste2e/common/k8sservice.py]: Fixed

File: teste2e/common/k8s.py (hunk 2 above)
Comment (phudtran) [reply teste2e/common/k8s.py]: Fixed

**Pull request merged**